Automattic/wp-calypso
# Atomic Hosting Panel: Add PHP Version switcher

#### Changes proposed in this Pull Request

Allow users to set the PHP version for their Business/eCommerce site.

#### Testing instructions

- Create an Atomic site (Business or eCommerce)
- Select the site
- Under manage pick "SFTP & MySQL" (might be renamed to "Hosting Configuration")
- See PHP Version panel
- It shows correct PHP version
- Try updating version
- Confirm everything looks ok

Screenshot:
<img width="653" alt="Screenshot 2019-11-22 at 18 03 04" src="https://user-images.githubusercontent.com/1103398/69445327-5dc6a400-0d52-11ea-8d32-05b8b0371a1a.png">
